test/mock/scripts/unload-recording-01.ts

```diff
@@ -15,6 +14,49 @@ import {
 import { basicMessageHandlers, basicBindings } from "../src/handlers";
 import { Page } from "@recordreplay/playwright";
 
+export function waitForTime(ms: number, waitingFor?: string) {
```

Comment:
:speech_balloon: Moved these from the old `harness.ts` file, since this was the only other place that needed them